appinventor/blocklyeditor/src/warningHandler.js

```diff
@@ -107,6 +111,39 @@ Blockly.WarningHandler.prototype.checkAllBlocksForWarningsAndErrors = function()
   }
 };
 
+Blockly.WarningHandler.prototype.warningNavPrevious = function() {
```

Comment:
Minor thing: I would prefer if these method names started with an action, e.g. gotoWarningNavPrevious, or maybe we can rename them in the form `previous[...]` so that code-wise the calls are `handler.previousWarning()`, `handler.nextError()`, etc.